src/python/pants/backend/python/util_rules/pex_from_targets_test.py

```diff
@@ -174,16 +174,16 @@ class RequirementMode(Enum):
     repository_pex__constraints = Mock()
 
     def assert_setup(
-        mode: RequirementMode,
+        _mode: RequirementMode,
         *,
-        internal_only: bool,
-        platforms: bool,
+        _internal_only: bool,
+        _platforms: bool,
```

Comment:
trying to figure out why these got an added `_` prefix.. ?

Reply:
They were colliding with variables in the outer scope.

Reply:
GH review has been horrible compared to alternatives since day 1 and continuing. It's been ... 7 years now? There's the can't whine about it free aspect, but there is also the convenience aspect. When we switched away from ReviewBoard, which was subjectively better, it was deemed the case that we were desperate enough for contributors that we had to reduce every possible friction and not be that odd project that forces you to use two sites.

Reply:
On the web interface (not the mobile app) there is a way to display the entire file in the diff. In the file header there is an expand arrow icon (line in the middle with arrows going up and down) you can click to expand the context. You can also expand individual sections by clicking the expand icon that takes the place of the skipped line numbers.

Reply:
Oh, cool. I've seen the individual expand buttons for each section but have overlooked the one to expand the entire file. Thanks 👍🏽 

And that's my reward for whining about it :)